team.go

```diff
@@ -57,50 +57,52 @@ func (s *Session) GetTeamWithOfficers(teamName string) (team Team, err error) {
 	return
 }
 
-// GetTeamHeadPositions retrieves all head-of-team positions for a given team ID.
+// getTeamPositions retrieves the positions for a given team ID and position type.
 // The amount of detail can be controlled using MyRadio mixins.
+// The position parameterType is either assistant or head
 // This consumes one API request.
-func (s *Session) GetTeamHeadPositions(id int, mixins []string) (head []Officer, err error) {
-	data, err := s.apiRequest(fmt.Sprintf("/team/%d/headpositions", id), mixins)
+func getTeamPositions(positionType string, id int, mixins []string, s *Session) (position []Officer, err error) {
+	var request string
+	switch positionType {
+	case "assistant":
```

Comment:
@LordAro wouldn't adding the if statement be just as inefficient as the switch case?